src/services/utils.js

```diff
@@ -211,7 +211,35 @@ export function keysToSnakeCase(collection) {
   return collection;
 }
 
-export default {
+/**
+ * Formats phone number with this mask: +55 555 555 5555
+ * @param countryCode {String} Selected country code
+ * @param phoneNumber {String} Phone number
+ * @returns {String} Formatted Phone number
+ */
+
+export function formatPhoneNumber(countryCode, phoneNumber) {
+  const countryCodeDigits = countryCode.length;
+  if (!phoneNumber) {
+    return '';
+  } else if (phoneNumber.length === countryCodeDigits) {
+    return countryCode;
+  } else if (phoneNumber.length > countryCodeDigits && phoneNumber.length <= countryCodeDigits + 3) {
+    const groupsOfNumbers = phoneNumber.substring(countryCodeDigits).match(/(\d{0,3})/);
+    return `${countryCode} ${groupsOfNumbers[1]}`;
+  } else if (phoneNumber.length > countryCodeDigits + 3 && phoneNumber.length <= countryCodeDigits + 6) {
+    const groupsOfNumbers = phoneNumber.substring(countryCodeDigits).match(/(\d{0,3})(\d{0,3})/);
+    return `${countryCode} ${groupsOfNumbers[1]} ${groupsOfNumbers[2]}`;
+  } else if (phoneNumber.length > countryCodeDigits + 6 && phoneNumber.length <= countryCodeDigits + 10) {
+    const groupsOfNumbers = phoneNumber.substring(countryCodeDigits).match(/(\d{0,3})(\d{0,3})(\d{0,4})/);
+    return `${countryCode} ${groupsOfNumbers[1]} ${groupsOfNumbers[2]} ${groupsOfNumbers[3]}`;
+  } else {
+    const groupsOfNumbers = phoneNumber.substring(countryCodeDigits).match(/(\d{0,3})(\d{0,3})(\d{0,4})(\d{0,10})/);
+    return `${countryCode} ${groupsOfNumbers[1]} ${groupsOfNumbers[2]} ${groupsOfNumbers[3]}${groupsOfNumbers[4]}`;
+  }
+}
```

Comment:
It might become very tricky to develop a formatted function from the stretch as it has many decisions that need to be discussed. What about using a **tiny** 3rd party library to manage formatting a phone number? For example, https://github.com/grantila/awesome-phonenumber#readme (I didn't check if it is the best option)

Reply:
Totally agree! specially now that we are going to add the zero for the format

Reply:
Actually the library is really good! I used it in other sections where I needed to validate the dial code, thanks for this suggestion 🥳